packages/pcd-collection/src/PCDCollection.ts

```diff
@@ -453,6 +455,34 @@ export class PCDCollection {
 
     return collection;
   }
+
+  /**
+   * Merges another PCD collection into this one.
+   * There are two options:
+   * - `filter` is a function used to filter out PCDs from the other
```

Comment:
Should document what "true" means (keep or discard) here.
Perhaps name the filter function "shouldInclude()" or something to imply which way the boolean goes.

Reply:
Agree w/ this

Reply:
It means the same as it does on `Array.filter`.